griptape/tools/base_tool.py

```diff
@@ -198,6 +198,7 @@ def install_dependencies(self, env: Optional[dict[str, str]] = None) -> None:
             cwd=self.tool_dir(),
             stdout=None if self.verbose else subprocess.DEVNULL,
             stderr=None if self.verbose else subprocess.DEVNULL,
+            check=False,
```

Comment:
Was adding `check=False,` intentional? `False` is the default ([docs](https://docs.python.org/3/library/subprocess.html#subprocess.run)), so this shouldn't be needed anyway...